rvgo/fast/memory.go

```diff
@@ -140,7 +140,7 @@ func (m *Memory) GetUnaligned(addr uint64, dest []byte) {
 	p, ok := m.pageLookup(pageIndex)
 	var d int
 	if !ok {
-		l := pageSize - pageAddr
+		l := PageSize - pageAddr
```

Comment:
It looks like `pageSize` and `PageSize` are different values though.

I see `pageSize = 1 << pageAddrSize = 1 << 10`
But `PageSize = 1 << PageAddrSize = 1 << 12` here: https://github.com/ethereum-optimism/asterisc/blob/056f92e40612fecd64a97184401c5120a75555e8/rvgo/fast/memory.go#L18C2-L18C10

Am I confusing comething?

Reply:
`pageSize` were incorrect remaining constants from some older code. `PageSize = 1 << 12` is the correct value across rvgo implementation and should be used instead.

Reply:
Got it ok - thanks for addressing